tests/plugins/table/_helpers/headersCorrectness.js

```diff
@@ -0,0 +1,22 @@
+/* exported assertHeadersCorrectnesssAfterManipulation */
+
+function assertHeadersCorrectnesssAfterManipulation( input, expected ) {
```

Comment:
I would rename it to `assertHeadersAfterManipulation` (btw, there is triple `s` in the name now), because `assert` on it's own indicates checking if something is correct.

Reply:
Also I see this method accepts only 2 parameters while in the most tests (3/4 files) you pass 3. The fact it works is that 2nd and 3rd parameter are always the same:D

So you may just use 2 params version. Or make it more explicit (and more readable), e.g. `assertHeadersAfterManipulation( inputHtmlId, expectedHtmlId, headers )`

Reply:
> I would rename it to `assertHeadersAfterManipulation`...

I had some enlightenment moment and I think the function name should be something like `testHeadersManipulation`. This function basically creates and runs entire test instead of only asserting headers so it should be named accordingly.